.github/workflows/solidity-tests.yml

```
@@ -0,0 +1,45 @@
+name: Solidity Tests CI
+
+on:
+  pull_request:
+    branches: [main, release/**]
+  push:
+    branches: [main, release/**]
+    tags: [v*]
+
+concurrency:
+  group: tests-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+  checks: write
+  pull-requests: write
+
+jobs:
+  BlockInfo:
+    name: BlockInfo Test Suite
+    uses: ./.github/workflows/test-workflow.yml
+    with:
+      testfilter: BlockInfo
```

Let's use a reusable tag
```suggestion
      testfilter: @solidityequiv
```

---

Done, but I still question putting all tests in one umbrella job.